manuscript/ha-docker-swarm/traefik.md

```diff
@@ -52,42 +52,40 @@ While it's possible to configure traefik via docker command arguments, I prefer
 Create `/var/data/traefik/traefik.toml` as follows:
 
 ```
-checkNewVersion = true
-defaultEntryPoints = ["http", "https"]
-
-# This section enable LetsEncrypt automatic certificate generation / renewal
-[acme]
-email = "<your LetsEncrypt email address>"
-storage = "acme.json" # or "traefik/acme/account" if using KV store
-entryPoint = "https"
-acmeLogging = true
-onDemand = true
-OnHostRule = true
-
-# Request wildcard certificates per https://docs.traefik.io/configuration/acme/#wildcard-domains
-[[acme.domains]]
-  main = "*.example.com"
-  sans = ["example.com"]
-
-# Redirect all HTTP to HTTPS (why wouldn't you?)
-[entryPoints]
-  [entryPoints.http]
+[global]
+  checkNewVersion = true
+
+# Enable the Dashboard
+[api]
+
+# Write out Traefik logs
+[log]
+  level = "DEBUG"
+  filePath = "/traefik.log"
```

Comment:
This is probably not what we want.

Reply:
No, that's the translation of the `--log.level=DEBUG` command-line option that was passed.